app/assets/stylesheets/finder_frontend.scss

```diff
@@ -209,6 +209,10 @@
 .facet-toggle {
   display: none;
   margin-bottom: govuk-spacing(6);
+
+  @include govuk-media-query($until: tablet) {
```

Comment:
Could we rewrite this to be mobile-first too? as the other `govuk-media-query` introduced with this commit? (as in `4` by default and `$from: tablet` to be `6`)?